PacketDotNet/LinkLayers.cs

```diff
@@ -75,23 +78,14 @@ public enum LinkLayers : byte
         /// <summary> FDDI </summary>
         Fddi = 10,
 
+        /// <summary> PPP over HDLC.</summary>
+        PppSerial = 50,
+
         /// <summary> LLC/SNAP encapsulated atm </summary>
-        AtmRfc1483 = 11,
+        AtmRfc1483 = 100,
 
         /// <summary> raw IP </summary>
-        Raw = 12,
-
-        /// <summary> BSD Slip.</summary>
-        SlipBSD = 15,
```

Comment:
is it ok to remove these? If they aren't correct then we should indicate that and remove them in their own commit, otherwise imo we should leave them in.